### PR TITLE
Migrated to weaviate

### PR DESCRIPTION
**Pull request by Bullrich**
Migrated library project to use Weaviate instead of Vectra.

### CODE REVIEW

File: src/routes/api/health/+server.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { weaviateClient } from '$lib/server/chatbot';
```
Comment (rzadp) [inline src/routes/api/health/+server.ts:1]: What's with the `+` in the filename? Is this intended?

File: package.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
 		"@typescript-eslint/eslint-plugin": "^6.7.5",
 		"@typescript-eslint/parser": "^6.7.5",
 		"autoprefixer": "^10.4.16",
+		"envsafe": "^2.0.3",
```
Comment (rzadp) [inline package.json:30]: It's used during runtime so I think it's a dependency, not a devDependency.

File: src/routes/api/health/+server.ts (hunk 1 above)
Comment (Bullrich) [reply src/routes/api/health/+server.ts:1]: I'm a bit confused, where is the +?

Comment (Bullrich) [reply src/routes/api/health/+server.ts:1]: AH I think it's the `+` from the git diff?

```diff
+ added this
- removed this
```

File: package.json (hunk 2 above)
Comment (Bullrich) [reply package.json:30]: It's actually used during setup, that's why I put it in devDependency. We don't need to build this with the final bundle

File: src/routes/api/health/+server.ts (hunk 1 above)
Comment (rzadp) [reply src/routes/api/health/+server.ts:1]: No no, I mean the `+server.ts` filename.

**Comment by rzadp** [comment]
I know nothing about Vectra or Weaviate. :D
Why is one used instead of the other?

File: src/routes/api/health/+server.ts (hunk 1 above)
Comment (Bullrich) [reply src/routes/api/health/+server.ts:1]: Ah, yes. That is a requirement for the routing. [Routing](https://kit.svelte.dev/docs/routing)

A rendered site has to be called `+page.svelte`, it's preloading logic `+page.ts` or `+page.server.ts`.

And if it is only an API endpoint, it needs to be called `+server.ts`.

In the directory name the route is defined.

So by having `routes/api/health/+server.ts` we are defining an API endpoint in `/api/health`.

Comment (rzadp) [reply src/routes/api/health/+server.ts:1]: Ah ok thanks. 👍

**Comment by Bullrich** [comment]
> I know nothing about Vectra or Weaviate. :D Why is one used instead of the other?

I'm gonna be migrating to Assistant API in the next PR, but:

- Both of them are Vector databases (split text into vectors and find the closest related text by vector approximation) [Related information](https://www.youtube.com/watch?v=klTvEwg3oJ4)
- Vectra works with a JSON file in the local repo.
  - This local file ends up having a size of 60+ mb, which takes way too much space for a small server instance.
- Weaviate works the same way, but it has its own database instead of a json file.
  - Migrating to Weaviate allows me to spin a database in a separate instance instead of having to update a JSON file continuosly.

**Review by rzadp — APPROVED** [review]

**Pull request merged**